inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSink.java

```diff
@@ -57,13 +47,10 @@ public void start() {
                 worker.start();
             }
         } catch (Exception e) {
-            LOG.error(e.getMessage(), e);
+            LOG.error("failed to start cls sink", e);
```

Comment:
It's better "LOG.error("failed to start cls sink,error:{}", e.getMessage(), e);".
Log4j will skip exception message in repeat exception.

Reply:
It's an optimization of JVM, uses **-XX:-OmitStackTraceInFastThrow** to forbid this option